src/app/personside/infotabs/ytelser/sykepenger/utbetalingerpåvent/utledUtbetalingPåVentÅrsak.test.ts

```diff
@@ -0,0 +1,159 @@
+import { utledUtbetalingPåVentÅrsak } from './utledUtbetalingerPåVentÅrsak';
+import { UtbetalingPåVent } from '../../../../../../models/ytelse/ytelse-utbetalinger';
+
+const utbetalingUtenPåventÅrsak: UtbetalingPåVent = {
+    vedtak: null,
+    utbetalingsgrad: 62,
+    oppgjørstype: 'Oppgjørstype',
+    arbeidskategori: null,
+    stansårsak: null,
+    ferie1: null,
+    ferie2: null,
+    sanksjon: null,
+    sykmeldt: null
+};
+
+describe('utleder riktig årsak for utbetaling på vent dersom', () => {
```

Comment:
I describe pleier man typisk å legge funksjonsnavnet som er fokuset av testen, typisk:

describe(lagGodteri)
   it(lager godteri om jeg er snill)
   it(lager ikke godteri om jeg er slem)

Småpirk,  men er ganske fin konvensjon synes jeg :)

Reply:
Synes også du godt kan skrive hva " riktig årsak" er i it().
f.eks:
``` 
describe(utledUtbetalingPåVentÅrsak)
    it(returnerer "'Inntektsopplysninger mangler'" når arbeidskategori er 99)
```
Slik kan man lese testene, og få en full beskrivelse av funksjonen, uten å trenge å lese logikken inni testen(som i blant er ganske krokit)

Reply:
👍